content/docs/reference/cmctl.md

```diff
@@ -355,3 +355,16 @@ these can be found on our [GitHub releases page](https://github.com/cert-manager
 In order to use the kubectl plugin you need its binary to be accessible under the name `kubectl-cert_manager` in your `$PATH`.
 
 You can run `kubectl cert-manager help` to test that the plugin is set up properly.
+
+### Auto-completion with the kubectl plugin
```

Comment:
On line 351 we say "While the kubectl plugin is supported, it is recommended to use `cmctl` as this enables a better experience via tab auto-completion."
Is this still true?

Reply:
No, this statement isn't true anymore since kubectl 1.26. Both the plugin and the standalone version come with auto-completion. I'll rephrase it.

Reply:
Thanks!